_assets/stylesheets/vendor/_normalize.scss

```diff
@@ -142,8 +144,8 @@ h1 {
  */
 
 mark {
-  background: #ff0;
   color: #000;
+  background: #ff0;
```

Comment:
Color literals like `#ff0` should only be used in variable declarations; they should be referred to via variable everywhere else.